tests/5.0/program_control/test_omp_target_offload.c

```diff
@@ -0,0 +1,100 @@
+//===--- test_omp_target_offload.c ----------------------------------------------------------------===//
+//
+// OpenMP API Version 5.0 Nov 2018
+//
+// This test checks for offloading behavior when OMP_TARGET_OFFLOAD is set to DEFAULT,
+// DISABLED, or MANDATORY. By DEFAULT, if the target device does not exist or the implementation
+// does not support the target device, the target code is executed on the host. Support for the
+// DISABLED policy is optional. OMP_TARGET_OFFLOAD is set to disabled, the behavior is as if 
+// no target device is avaiable and target regions execute on the host. When set to Mandatory, 
+// the program will terminate execution when a target construct is encountered and a target device
+// is not available or supported by the implementation. 
+// 
+// This tet was adopted from OpenMP 5.0 Examples Doc -> target_offload_control.1.c 
```

Comment:
Typo: test 
Only add this line if most of the code was taken from the examples.